spec/spec_helper.rb

```diff
@@ -1,12 +1,25 @@
 $LOAD_PATH.unshift File.expand_path("../../lib", __FILE__)
 
+# Needs to be required and started before danger
+require "simplecov"
+SimpleCov.start do
+  add_filter "/spec/"
+end
+
 require "danger"
 require "webmock"
 require "webmock/rspec"
 require "json"
 
+require "support/gitlab_helper"
+require "support/github_helper"
+
 RSpec.configure do |config|
   config.filter_gems_from_backtrace "bundler"
+  config.include Danger::Support::GitLabHelper, host: :gitlab
+  config.include Danger::Support::GitHubHelper, host: :github
```

Comment:
I've reverted back to this method of controlling the environment in tests. I believe it's much cleaner and more in line with rspec